pkg/dirpublisher/dirpublisher.go

```diff
@@ -17,26 +17,33 @@ import (
 	"github.com/ipfs/interface-go-ipfs-core/options"
 	"github.com/ipld/go-car"
 	"github.com/tablelandnetwork/rigs/pkg/nftstorage"
+	"github.com/tablelandnetwork/rigs/pkg/storage/local"
 	"github.com/tablelandnetwork/rigs/pkg/wpool"
 	"golang.org/x/time/rate"
 )
 
 // DirPublisher publishes a directory to nft.storage.
 type DirPublisher struct {
+	localStore local.Store
```

Comment:
Little refactor so that the dir publisher writes to the local db as soon as it adds a folder of images to ipfs.